src/client/java/minicraft/core/Renderer.java

```diff
@@ -544,24 +545,24 @@ private static void renderFocusNagger() {
 		int h = 1;
 
 		// Renders the four corners of the box
-		screen.render(xx - 8, yy - 8, 0, 6, 0, hudSheet.getSheet());
-		screen.render(xx + w * 8, yy - 8, 0, 6, 1, hudSheet.getSheet());
-		screen.render(xx - 8, yy + 8, 0, 6, 2, hudSheet.getSheet());
-		screen.render(xx + w * 8, yy + 8, 0, 6, 3, hudSheet.getSheet());
+		screen.render(xx - 8, yy - 8, 0, 6, 0, hudSheet.get());
+		screen.render(xx + w * 8, yy - 8, 0, 6, 1, hudSheet.get());
+		screen.render(xx - 8, yy + 8, 0, 6, 2, hudSheet.get());
+		screen.render(xx + w * 8, yy + 8, 0, 6, 3, hudSheet.get());
 
 		// Renders each part of the box...
 		for (int x = 0; x < w; x++) {
-			screen.render(xx + x * 8, yy - 8, 1, 6, 0, hudSheet.getSheet()); // ...Top part
-			screen.render(xx + x * 8, yy + 8, 1, 6, 2, hudSheet.getSheet()); // ...Bottom part
+			screen.render(xx + x * 8, yy - 8, 1, 6, 0, hudSheet.get()); // ...Top part
+			screen.render(xx + x * 8, yy + 8, 1, 6, 2, hudSheet.get()); // ...Bottom part
 		}
 		for (int y = 0; y < h; y++) {
-			screen.render(xx - 8, yy + y * 8, 2, 6, 0, hudSheet.getSheet()); // ...Left part
-			screen.render(xx + w * 8, yy + y * 8, 2, 6, 1, hudSheet.getSheet()); // ...Right part
+			screen.render(xx - 8, yy + y * 8, 2, 6, 0, hudSheet.get()); // ...Left part
+			screen.render(xx + w * 8, yy + y * 8, 2, 6, 1, hudSheet.get()); // ...Right part
```

Comment:
Is is a clever an logical name change.

Reply:
I am not sure what you mean. It is changed from `LinkedSprite#getSheet()` into `Supplier<MinicraftImage>#get()`.

Reply:
It is a good change.